packages/react-post/lib/react-post.js

```diff
@@ -186,6 +186,7 @@ const ReadMore = styled.p`
 	line-height: 18px !important;
 	font-weight: 500 !important;
 	letter-spacing: -0.2px !important;
+	margin-bottom: 13px;
```

Comment:
@creador-dev adding margin is not the best solution. You must think about it as a dynamic component and if something changes in the paragraph everything will get misaligned again.